Components/Button/button.css

```diff
@@ -0,0 +1,131 @@
+@import url("/CSS/main.css");
+
+.btn {
+  outline: none;
+  border: none;
+  padding: 0.9rem;
+  text-transform: uppercase;
+  font-size: 1rem;
+  margin: 0.3rem;
+  border-radius: 0.4rem;
+}
+
+.btn-solid {
+  background-color: var(--primary-color);
+  color: var(--white);
+  cursor: pointer;
+}
+.btn-icon:hover,
+.btn-solid:hover,
+.btn-float:hover {
+  background-color: var(--primary-color-hover);
+}
+.btn-outline {
+  border: 2px solid var(--primary-color);
+  background-color: transparent;
+  cursor: pointer;
+}
+.btn-outline:hover {
+  background-color: var(--primary-color);
+  color: var(--white);
+}
+.btn-icon {
+  background-color: var(--primary-color);
+  color: var(--white);
+  cursor: pointer;
+}
+.btn-link {
+  border-radius: 0;
+  background-color: transparent;
+  border-bottom: 2px solid var(--primary-color);
+  cursor: pointer;
+}
+.btn-link:hover {
+  background-color: var(--light-grey);
+  border: radius 0.4rem;
+}
+
+.disabled-btn-solid,
+.disabled-btn-icon {
+  background-color: var(--primary-color-hover);
+  color: var(--white);
+}
+.disabled-btn-outline {
+  outline: 2px solid var(--primary-color-hover);
+}
+
+.disabled-btn-link {
+  border-radius: 0;
+  border-bottom: 2px solid var(--primary-color-hover);
+}
+.disabled-btn-link a {
+  cursor: default;
```

Comment:
it will be good if u add only one class btn-disable for all button 
what u can do if just make cursor: default and filter: brightness(30%); like that or u can use opacity too

Reply:
Thank you for the suggestion, I will change it.